### PR TITLE
update stack version

### PR DESCRIPTION
**Pull request by ismailmustafa**

### CODE REVIEW

File: bin/release [hunk 1]
```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env bash
 
+stack exec quillio
```
Comment (mfine) [inline bin/release]: Can you remove this?

**Comment by ismailmustafa** [comment]
Oops, sorry about that. Just reverted the last two commits.

**Comment by mfine** [comment]
Thanks! and thanks for the update! (note, you can also effect the update without by setting `STACK_VER` as a env var for compile time).

**Comment by ismailmustafa** [comment]
Ah that's good to know! Thank you.

**Pull request merged**